pkg/graphql/request.go

```diff
@@ -76,8 +76,17 @@ func (r *Request) Normalize(schema *Schema) (result NormalizationResult, err err
 	return NormalizationResult{Successful: true, Errors: nil}, nil
 }
 
-func (r Request) CalculateComplexity(complexityCalculator ComplexityCalculator) int {
-	return 1
+func (r *Request) CalculateComplexity(complexityCalculator ComplexityCalculator, schema *Schema) (nodeCount, complexity, depth int, err error) {
```

Comment:
Maybe we can provide a `ComplexityResult` struct or something like that. 4 return values can be confusing as you have to remember the order.